app/src/screenshots/res/values/themes.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <style name="AppThemeBase" parent="BpkTheme">
+    <item name="bpkFontFamilyBase">@font/skyscanner_relative_android_book</item>
+    <item name="bpkFontFamilyEmphasized">@font/skyscanner_relative_android_bold</item>
+    <item name="android:lineSpacingMultiplier">1</item>
+    <item name="android:overScrollMode">never</item>
```

Comment:
This is the most important stuff for screenshots consistency

Reply:
This was consistent before this change by using the `waitForIdleSync` for the nav bar. I still think something's not right here with how things are set up that cause these issues to happen as that used to be an issue in the past, but is no longer happening.

Reply:
It was not. The screenshots were changing all the time.

`waitForIdleSync()` only waited until the scrolling was done – but not the overscroll effect. The amount of overscroll effect (which depends on time, velocity, and direction) was always slightly different and usually, it was only the pixels at the overscroll effect border which was slightly different.